src/main/kotlin/screeps/api/Constants.kt

```diff
@@ -289,3 +305,38 @@ val TEXT_ALIGN_CENTER: TextAlignConstant = "center".unsafeCast<TextAlignConstant
 val TEXT_ALIGN_RIGHT: TextAlignConstant = "right".unsafeCast<TextAlignConstant>()
 val ALGORITHM_ASTAR: AlgorithmConstant = "astar".unsafeCast<AlgorithmConstant>()
 val ALGORITHM_DIJKSTRA: AlgorithmConstant = "dijkstra".unsafeCast<AlgorithmConstant>()
+
+
+external interface PowerClassConstant : StringConstant
+external interface PowerEffectConstant : IntConstant
+
+enum class POWER_CLASS(val value: PowerClassConstant) {
```

Comment:
as you discussed with Jomik this should probably be removed in favour of a 'classic' constant